packages/vite/src/node/server/index.ts

```diff
@@ -563,20 +569,36 @@ export async function _createServer(
       return devHtmlTransformFn(server, url, html, originalUrl)
     },
     async ssrLoadModule(url, opts?: { fixStacktrace?: boolean }) {
+      warnFutureDeprecation(config, 'ssrLoadModule')
       return ssrLoadModule(url, server, undefined, opts?.fixStacktrace)
     },
-    async ssrFetchModule(url: string, importer?: string) {
-      return ssrFetchModule(server, url, importer)
-    },
     ssrFixStacktrace(e) {
-      ssrFixStacktrace(e, moduleGraph)
+      ssrFixStacktrace(e, server.environments.ssr.moduleGraph)
     },
     ssrRewriteStacktrace(stack: string) {
-      return ssrRewriteStacktrace(stack, moduleGraph)
+      return ssrRewriteStacktrace(stack, server.environments.ssr.moduleGraph)
```

Comment:
Agreed. The idea is to avoid deprecations warnings for current methods in Vite 6, and then add them in a future 6.x minor (at the same time we remove the experimental label from Environment API).